src/NHibernate.TestDatabaseSetup/NHibernate.TestDatabaseSetup.csproj

```diff
@@ -7,19 +7,24 @@
     <IsTestProject>true</IsTestProject>
     <NoWarn>$(NoWarn);3001;3002;3003;3005</NoWarn>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+  <PropertyGroup Condition="$(NhNetCoreApp)">
     <OutputType>Exe</OutputType>
     <GenerateProgramFile>false</GenerateProgramFile>
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\NHibernate.Test\NHibernate.Test.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="NUnit" Version="3.12.0" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.13.0" />
+    <PackageReference Include="NUnit" Version="3.13.2" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
-    <PackageReference Include="NUnitLite" Version="3.12.0" />
+  <ItemGroup Condition="$(NhNetFx)">
```

Comment:
This and 2 following groups are confusing, I'm not sure I understand the intention.

Reply:
netcoreapp2.0 cannot compile with a reference to NUnit3TestAdapter. So it has to be referenced only for netFx and 2.1.

Maybe I should not mix $(NhNetCoreApp) references (for both 2.0 and 2.1) and explicit 2.1 references, for easing understanding what is done.

Or even maybe I could just drop the NUniteLite reference for 2.0 too, I have not checked.

Reply:
I cannot remove NUniteLite reference for netcoreap2.0, or else I would have to use a `NETCOREAPP2_1` only condition in test program files, instead of using the `NETCOREAPP2_0_OR_GREATER` condition already in master.

So I have changed for using explicite target framework references for netcore, which duplicates quite a few references for the main test project.